src/manipulation.js

```diff
@@ -201,7 +201,7 @@ jQuery.extend({
 		return clone;
 	},
 
-	buildFragment: function( elems, context, scripts, selection ) {
+	buildFragment: function( elems, context, scripts, selection, ignored ) {
```

Comment:
The burden of proof is on the person that wants to introduce a change, i.e. here it's you. ;)

I just said it *might* introduce a perf penalty, I don't know.